modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectDefinitionAPI.macro

```diff
@@ -859,12 +857,19 @@ definition {
 	}
 
 	macro relateObjectEntries {
```

Comment:
It's good we can have not too many codes. But I think it's better to have two different methods for PUT and DELETE, since DELETE is kind of breakObjectEntries not relate them. :smile:

Reply:
Thanks guys, I will separate them.